test/python/test_select.py

```diff
@@ -104,3 +104,58 @@ def run_test(sess):
 
         assert (
             self.with_ngraph(run_test) == self.without_ngraph(run_test)).all()
+
+
+class TestWhere(NgraphTest):
+    env_map = None
+
+    def __init__(self):
+        self.store_env_variables(['TF_OV_CONSTANT_FOLDING'])
+        self.set_env_variable('TF_OV_CONSTANT_FOLDING', '1')
+
+    def __del__(self):
+        self.restore_env_variables(env_map)
+
+    def test_where(self):
+        a = np.random.random(size=[2, 2]).astype(np.float32)
```

Comment:
May be better to not randomize the test data, to ensure there is a predictable value for the number of '3'

Reply:
Fixed in #742